### ISSUE TITLE
SpecsFileRunner does not work under Scala 2.8 nightly builds

### ISSUE THREAD

**Issue by GoogleCodeExporter** [etorreborre/specs#75]
```
Under Scala 2.8 and Specs 1.5.0 or Specs 1.4.4, SpecsFileRunner reports
"0 example, 0 expectation, 0 failure, 0 error"

Under Specs 1.4.3, it reports
"Could not load ExampleSpec$" 

where "ExampleSpec$" is the spec object in the attached example.

All work fine under Scala 2.7.4.

The Makefile has instructions for running with the different versions of specs.
```

Original issue reported on code.google.com by `deanwamp...@gmail.com` on 21 May 2009 at 2:31

Attachments:
- [specs-scala28-runner-bug.zip](https://storage.googleapis.com/google-code-attachments/specs/issue-75/comment-0/specs-scala28-runner-bug.zip)

**Comment by GoogleCodeExporter** [comment]
```
Hi Dean,

This is well possible that specs needs to be recompiled with Scala 2.8 in order 
to 
really work with Scala 2.8. I'm indeed expecting some issues given the changes 
in 
2.8. I experimented similar issues with some previous Scala upgrades.

Unfortunately, I can't do better at the moment (just relocated to another 
country, so 
I'm very busy) than advise you to checkout the trunk and use Maven to recompile 
everything against 2.8 and see how it goes.

E.
```

Original comment by `etorrebo...@gmail.com` on 21 May 2009 at 3:31
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
The project will be recompiled with 2.8 and published when 2.8 is official.
```

Original comment by `etorrebo...@gmail.com` on 23 May 2009 at 11:30
- Changed state: **WontFix**
